src/routes/HomePage/HomePage.js

```diff
@@ -1,11 +1,29 @@
-import React from 'react';
-import { Section } from '../../components/Utils/Utils';
+import React, { useEffect, useState } from 'react';
+import { Section, SecondaryButton } from '../../components/Utils/Utils';
```

Comment:
I think `SecondaryButton` is unused and can be removed

Reply:
removed